docs/conf.py

```diff
@@ -34,35 +34,42 @@
 source_suffix = ".rst"
 master_doc = "index"
 project = "21cmSense"
-year = "2019"
+year = str(datetime.now().year)
 author = "Jonathan Pober and Steven Murray"
 copyright = "{0}, {1}".format(year, author)
-version = release = "2.0.0.alpha"
+version = release = __version__
 templates_path = ["templates"]
 
 pygments_style = "trac"
 extlinks = {
     "issue": ("https://github.com/steven-murray/21cmSense/issues/%s", "#"),
     "pr": ("https://github.com/steven-murray/21cmSense/pull/%s", "PR #"),
 }
-# on_rtd is whether we are on readthedocs.org
-on_rtd = os.environ.get("READTHEDOCS", None) == "True"
 
-if not on_rtd:  # only set the theme if we're building docs locally
-    html_theme = "sphinx_rtd_theme"
+html_theme = "furo"
 
 html_use_smartypants = True
 html_last_updated_fmt = "%b %d, %Y"
 html_split_index = False
-html_sidebars = {"**": ["searchbox.html", "globaltoc.html", "sourcelink.html"]}
-html_short_title = "%s-%s" % (project, version)
+
+# html_sidebars = {
+#     "**": [
+#         "sidebar/scroll-start.html",
+#         "sidebar/brand.html",
+#         "sidebar/search.html",
+#         "sidebar/navigation.html",
+#         "sidebar/ethical-ads.html",
+#         "sidebar/scroll-end.html",
+#     ]
+# }
```

Comment:
are we keeping this around just in case or can we delete it?

Reply:
I'm still working on it --- I realized the docs are not showing the sidebar all of a sudden. Will remove this entirely if it's not required.

Reply:
okie dokie :+1: wasn't sure if it was done done or not!